### PR TITLE
[#57593] 14.5 Documentation: Release custom field columns/filters in …

### PR DESCRIPTION
**Pull request by MayaBerd**
…project lists to Community Edition  https://community.openproject.org/work_packages/57593

[#57593] 14.5 Documentation: Release custom field columns/filters in project lists to Community Edition

https://community.openproject.org/work_packages/57593

# Ticket

# What are you trying to accomplish?

## Screenshots

# What approach did you choose and why?

# Merge checklist

- [ ] Added/updated tests
- [ ] Added/updated documentation in Lookbook (patterns, previews, etc)
- [ ] Tested major browsers (Chrome, Firefox, Edge, ...)

### CODE REVIEW

File: docs/user-guide/projects/project-lists/Name-private-projects-filter.png
Comment (marcalcobe) [inline docs/user-guide/projects/project-lists/Name-private-projects-filter.png:1]: The "x" close icon is now only visible when there is something input filled. I don't know if we want to update the screenshots here as there is nothing added in the search.

File: docs/user-guide/projects/project-lists/activity-global-menu.png
Comment (marcalcobe) [inline docs/user-guide/projects/project-lists/activity-global-menu.png:1]: Same applies here with the "X" in the search.

File: docs/user-guide/projects/project-lists/configure-view-project-list.png
Comment (marcalcobe) [inline docs/user-guide/projects/project-lists/configure-view-project-list.png:1]: Same applies here with the "X" in the search.

File: docs/user-guide/projects/project-lists/expand-link-project-description.png
Comment (marcalcobe) [inline docs/user-guide/projects/project-lists/expand-link-project-description.png:1]: Same applies here with the "X" in the search.

File: docs/user-guide/projects/project-lists/export-projects.png
Comment (marcalcobe) [inline docs/user-guide/projects/project-lists/export-projects.png:1]: Same applies here with the "X" in the search.

File: docs/user-guide/projects/project-lists/name-new-project-list.png
Comment (marcalcobe) [inline docs/user-guide/projects/project-lists/name-new-project-list.png:1]: Same applies here with the "X" in the search.

File: docs/user-guide/projects/project-lists/new-subproject-project-list.png
Comment (marcalcobe) [inline docs/user-guide/projects/project-lists/new-subproject-project-list.png:1]: Same applies here with the "X" in the search.

File: docs/user-guide/projects/project-lists/open-as-gantt-view.png
Comment (marcalcobe) [inline docs/user-guide/projects/project-lists/open-as-gantt-view.png:1]: Same applies here with the "X" in the search.

File: docs/user-guide/projects/project-lists/overall-activity-link.png
Comment (marcalcobe) [inline docs/user-guide/projects/project-lists/overall-activity-link.png:1]: Same applies here with the "X" in the search.

File: docs/user-guide/projects/project-lists/openproject_user_guide_project_lists.png
Comment (marcalcobe) [inline docs/user-guide/projects/project-lists/openproject_user_guide_project_lists.png:1]: Same applies here with the "X" in the search.

File: docs/user-guide/projects/project-lists/private-project-filter-rename-delete.png
Comment (marcalcobe) [inline docs/user-guide/projects/project-lists/private-project-filter-rename-delete.png:1]: Same applies here with the "X" in the search.

File: docs/user-guide/projects/project-lists/private-project-filter-saved.png
Comment (marcalcobe) [inline docs/user-guide/projects/project-lists/private-project-filter-saved.png:1]: Same applies here with the "X" in the search.

File: docs/user-guide/projects/project-lists/project-lists-share-with-everyone-list.png
Comment (marcalcobe) [inline docs/user-guide/projects/project-lists/project-lists-share-with-everyone-list.png:1]: Same applies here with the "X" in the search.

File: docs/user-guide/projects/project-lists/projects-lists-default-filters.png
Comment (marcalcobe) [inline docs/user-guide/projects/project-lists/projects-lists-default-filters.png:1]: Same applies here with the "X" in the search.

File: docs/user-guide/projects/project-lists/star-yellow-project-list.png
Comment (marcalcobe) [inline docs/user-guide/projects/project-lists/star-yellow-project-list.png:1]: Same applies here with the "X" in the search.

File: docs/user-guide/projects/project-lists/star-project-list.png
Comment (marcalcobe) [inline docs/user-guide/projects/project-lists/star-project-list.png:1]: Same applies here with the "X" in the search.

File: docs/user-guide/projects/project-lists/save-button-filtered-view.png
Comment (marcalcobe) [inline docs/user-guide/projects/project-lists/save-button-filtered-view.png:1]: Same applies here with the "X" in the search.

**Review by marcalcobe — CHANGES_REQUESTED** [review]
Hi @MayaBerd,

I've added a lot of repetitive comments, sorry about that! The only thing I detected is that the "X" to clear the search bar input even if there is no input. This is fixed in the BUGs mentioned in the activity in https://community.openproject.org/projects/documentation/work_packages/55234/.

I don't know if its really relevant to the docs as it still exemplifies the feature. I don't really mind having it as it is right now.

**Comment by MayaBerd** [comment]
Thanks for the review @marcalcobe !

Fixing of the X was moved to 14.6 (https://community.openproject.org/projects/openproject/work_packages/57777), so it is still visible for now. I'll update the screenshots again for 14.6

> Hi @MayaBerd,
> 
> I've added a lot of repetitive comments, sorry about that! The only thing I detected is that the "X" to clear the search bar input even if there is no input. This is fixed in the BUGs mentioned in the activity in https://community.openproject.org/projects/documentation/work_packages/55234/.
> 
> I don't know if its really relevant to the docs as it still exemplifies the feature. I don't really mind having it as it is right now.

**Pull request merged**